src/js/BookReader/Mode2Up.js

```diff
@@ -1170,7 +1170,9 @@ export class Mode2Up {
     const { currentIndexL, currentIndexR } = this.br.twoPage;
     const ADJACENT_PAGES_TO_LOAD = 3;
 
-    let lowPage = book.getPage(min(currentIndexL, currentIndexR));
+    // currentIndexL can be -1; getPage returns the last page of the book
```

Comment:
q: is `-1` the value for the pair of either front or back cover? I have seen the front cover spread as this[-1, 0].  Just curious if the last page is like this as well [100, -1]?

Reply:
In mode 2 up, -1 denotes the start [-1,0]. However when I implemented getPage, I made the decision that getPage(-1) should return the back cover :P Forgot that Mode2up did this! So now there are like competing definitions for what -1 means. Not the end of the world, but it would be nice to make these consistent at some point.